ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -108,6 +108,7 @@ pipeline {
 
         stage('Cluster Bootstrap') {
             steps {
+                sh(script: 'skuba/ci/infra/testrunner/testrunner --platform ${PLATFORM} ssh_all -r master "sudo mount -t tmpfs -o size=1048M tmpfs /var/lib/etcd"', label: 'ramfs for etcd dirs')
```

Comment:
1Gb of ram for etcd is not excessive?

Reply:
arent you sick? :P

Its just WIP, I dont even know if this will work. Will redo it later with a more proper size *if* this work and its a decent stopgap for our current issues with etcd